formatters/vectorTraffic.js

```diff
@@ -42,6 +51,7 @@ module.exports = function vectorTraffic(fields) {
     fsv.cleanInt(fields.status, 2),
     fields.cacheHit === 'HIT',
```

Comment:
does it make sense to remove the cacheHit field? or should we leave it in for simplicity in deployment and consistency with pelias_traffic_v2?